.github/workflows/main.yml

```
@@ -46,5 +49,8 @@ jobs:
           HOSTNAME: localhost
           STATICFILES_STORAGE: django.contrib.staticfiles.storage.StaticFilesStorage
           DEFAULT_FILE_STORAGE: django.core.files.storage.FileSystemStorage
+          DEFAULT_FROM_EMAIL: test@test.com
```

```suggestion
          DEFAULT_FROM_EMAIL: test@example.com
```
Pour les domaines de test j'utiliserais un des [domaines réservés](https://en.wikipedia.org/wiki/Example.com).